src/analysis/retail/evoker/augmentation/modules/talents/Accretion.tsx

```diff
@@ -91,6 +91,9 @@ class Accretion extends Analyzer {
     if (event.ability.guid === SPELLS.UPHEAVAL_DAM.id) {
       this.totalUpheavalDamage += event.amount + (event.absorbed ?? 0);
     }
+    if (event.ability.guid === SPELLS.UPHEAVAL_DOT.id) {
+      this.totalUpheavalDamage += event.amount + (event.absorbed ?? 0);
+    }
```

Comment:
No need for separate if statement here.
```suggestion
    if (event.ability.guid === SPELLS.UPHEAVAL_DAM.id || event.ability.guid === SPELLS.UPHEAVAL_DOT.id) {
      this.totalUpheavalDamage += event.amount + (event.absorbed ?? 0);
    }
```